proto/money/rpc/master_folio_svc.proto

```diff
@@ -40,10 +41,24 @@ message MasterFolioServiceRemoveMasterFolioResponse {
 	ReservationFolioSvcRemoveMasterFolioResult result = 1;
 }
 
+enum MasterFolioSvcFolioSwapResult {
+	MASTER_FOLIO_SVC_FOLIO_SWAP_OK = 0;
+	MASTER_FOLIO_SVC_FOLIO_SWAP_IVALID_NO_MASTER_FOLIO = 1;
+	MASTER_FOLIO_SVC_FOLIO_SWAP_INVALID = 2;
```

Comment:
Agree on the "Catch all" point. I think you should call it "unknown error" though or something a bit different. (1) Unknown or "other" makes it clear this is a catch-all, and (2) this seems too similar to the one above it.